src/main/java/com/customer/CustomerResponse/CustomerResponse.java

```diff
@@ -29,16 +34,17 @@ public void setStatus(Date joiningDate, Date expiryDate) {
 		Date experienceDate = c.getTime();
 
 		if ((currentDate.after(expiryDate)))
-			status = "Expired";
+			status = Status.EXPIRED;
 		else if (currentDate.after(experienceDate))
-			status = "Experienced";
+			status = Status.EXPERIENCED;
 		else if (currentDate.before(experienceDate))
-			status = "New";
+			status = Status.NEW;
 		else
-			status = "Invalid Status";
+			status = Status.INVALID;
+		return status;
 	}
 
-	public String getStatus() {
+	public Status getStatus() {
```

Comment:
Why do you need getStatus ? The getter should already be there with @Data annotation